src/testing/test-helpers.ts

```diff
@@ -0,0 +1,22 @@
+import { AxePuppeteer } from '@axe-core/puppeteer';
+
+const ignoredDefaults = ['document-title', 'html-has-lang', 'page-has-heading-one', 'landmark-one-main', 'region'];
```

Comment:
These ignored rules don't make much sense when we are looking at an individual component. They seem more useful with an entire page in mind.

Reply:
1. This sounds good :+1: 
2. This line is super long; we should probably merge / cherry-pick the prettier / eslint changes I made in `master` into this branch :wink:

Reply:
Those changes are merged in, but nothing happens when I save the file. Perhaps we are missing a config to limit line length?